ui/src/reusable_ui/components/card_select/CardSelectList.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {PureComponent, ReactElement} from 'react'
+import {ErrorHandling} from 'src/shared/decorators/errors'
+
+import {CardSelectCardProps} from 'src/types/cardSelect'
+
+interface Props {
+  children: Array<ReactElement<CardSelectCardProps>>
+  legend: string
+}
+
+@ErrorHandling
+class CardSelectList extends PureComponent<Props> {
+  public render() {
+    const {children, legend} = this.props
+
+    return (
+      <fieldset className="card-select--wrapper">
```

Comment:
Curious about your use of `fieldset` here, isn't that usually for grouping items in a form?

Reply:
Yes, this is to create an accessible grouping of the checkboxes when used in a form.